### PR TITLE
"GI-120-db-aufsetzen"

### PR DESCRIPTION
**Pull request by MerlindCPohl**
## Description

Please describe the purpose of this Pull Request.  
Explain what has been changed and why these changes are necessary.

> **Reminder:** The Jira ticket **must be linked in the PR title**  
> Example: `GI-21: Implement user login flow`

## Jira Ticket
Link to the Jira ticket: [GI-XXX](https://techstudyfinder.atlassian.net/browse/GI-XXX)

## Author Checklist

- [x] Code follows project coding standards and conventions
- [x] Tests have been written or updated
- [x] All tests pass locally
- [x] No sensitive data or secrets are included
- [x] Documentation (README, comments) has been updated if necessary

## Reviewer Notes

Please verify the following during review:

- The implementation matches the described requirements and Jira ticket
- Code changes are logically correct and maintainable
- No unnecessary dependencies or side effects introduced
- Tests cover critical logic and edge cases
- Naming, structure, and readability meet team standards

## Additional Context (optional)

Add any background information or references that help reviewers understand the changes.

---

_Please ensure all criteria are met before merging._

### CODE REVIEW

**Comment by DamarisLi** [comment]
Danke dir für's Aufsetzen! Denk bitte nächstes Mal daran, die PR description auszufüllen :)

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        const result = await pool.query('SELECT NOW()');
+        res.json({ success: true, time: result.rows[0].now });
+    } catch (err) {
+        console.error('Datenbankfehler:', err);
+        res.status(500).json({ success: false, error: err.message || err });
+    }
+});
+
+const PORT = process.env.PORT || 3001;
+app.listen(PORT, () => console.log(`Server läuft auf Port ${PORT}`));
```
Comment (DamarisLi) [inline server/server.js]: Das hier gibt leider einen Konflikt wenn Frontend und Backend schon laufen, weil hier nochmal das Backend gestartet wird. Ich glaube db.js kann einfach in der index.ts hinzugefügt werden, wenn ich mich nicht irre

**Comment by MerlindCPohl** [comment]
> Danke dir für's Aufsetzen! Denk bitte nächstes Mal daran, die PR description auszufüllen :)

Alles klar, dachte Beschreibung im PR reicht für dieses Thema, but sure :)

**Review by DamarisLi — APPROVED** [review]
Super, danke!

**Review by johanna2402 — APPROVED** [review]
